### DISCUSSION TITLE
How to restor pod backup from full ns backup

### DISCUSSION THREAD

**Discussion by tariq1278** [vmware-tanzu/velero#3325] [Community support Q&A] [answered]
Hello folks,

I have taken full ns backup now I want to take restore only pod backup from full ns backup ..how to proceed...or we have to restore from full backup...

**Comment by zubron** [comment, marked as answer]
Hi @tariq1278! Please see our documentation on [resource filtering](https://velero.io/docs/v1.5/resource-filtering/). It is not possible to specify a specific pod to restore by name, but you can specify that only specific resource types, or resource with particular labels are restored.
